hw1-app/src/app/hwork1/search/search.component.ts

```diff
@@ -10,17 +12,24 @@ export class SearchComponent implements OnInit {
   @Output() searchStringOnChange = new EventEmitter<string>();
 
   public searchString: string;
+  public subject = new Subject<string>();
```

Comment:
Not very good naming for this variable, also it's good approach to add  $ symbol in the end of observables/subject names.

Reply:
Will rename it